import logging

logging.error('Это лог какой-то ошибки')